src/Adapter/Order/CommandHandler/IssueStandardRefundHandler.php

```diff
@@ -0,0 +1,154 @@
+<?php
```

Comment:
On new files, please don't forget  `declare(strict_types=1);`

Reply:
You're right I never think about it But I don't want to lose the approved status just for this
That said maybe we could make a script to automatically add this in all the new files in 177?!